### PR TITLE
Fixes #14353: Remove unencrypted body files

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/14353

### CODE REVIEW

File: techniques/system/common/1.0/update.cf [hunk 1]
```diff
@@ -25,11 +24,9 @@
 # Use for sensitive files
 body copy_from remote(server, path)
 {
-
         servers    => {
           "${server}"
         };
-        encrypt    => "true";
```
Comment (peckpeck) [inline techniques/system/common/1.0/update.cf]: Is that the default ?

**Comment by amousset** [comment]
Commit modified

File: techniques/system/common/1.0/update.cf (hunk 1 above)
Comment (amousset) [reply techniques/system/common/1.0/update.cf]: Good spot it is not actually. This has practically no impact as "When used with protocol_version 2 or greater this attribute is a noop as the entire session is encrypted." but who knows, it's probaly better to keep it anyway.

File: techniques/system/common/1.0/update.cf [hunk 2]
```diff
@@ -74,30 +51,9 @@ body copy_from copy_digest_without_perms(from)
         portnumber => "${system_common.community_port}";
 }
 
-# Use for non-sensitive policy files
-body copy_from remote_unsecured_without_perms(server, path)
```
Comment (ncharles) [inline techniques/system/common/1.0/update.cf]: this is used in techniques/system/distributePolicy/1.0/propagatePromises.st - you should remove it there also

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
OK, squash merging this PR

**Pull request merged**